src/core/server/docs/kib_core_logging.mdx

```diff
@@ -323,6 +323,83 @@ The maximum number of files to keep. Once this number is reached, oldest files w
 
 The default value is `7`
 
+**deprecated**: Please use a retention policy instead.
+
+#### Retention Policies
+
+The retention policy defines when the rolled files must be deleted.
+
+##### maxFiles
+
+Retention policy that will keep, at max, the defined number of files (ordered from the most recent to the oldest one)
+and delete the others. 
+
+This is the same behavior as what was previously achieved via the deprecated `strategy.max` field.
+
+**Example**: Appender configured to keep `5` rolled files:
+```yaml
+logging:
+  appenders:
+    rolling-file:
+      type: rolling-file
+      fileName: /var/logs/kibana.log
+      layout:
+        type: pattern
+      policy:
+        // ...
+      strategy:
+        // ...
+      retention:
+        maxFiles: 5
+```
+
+##### maxAccumulatedFileSize
+
+Retention policy based on the size on disk used by the rolled files. Size count will be
+performed from the most recent file to the oldest one, and will delete all files
+(including the one breaching the threshold) over the limit.
+
+Note that only the size from the rolled files is taken into account (the active file is ignored).
+
+**Example**: Appender configured to keep 5GB of rolled logs at maximum:
+```yaml
+logging:
+  appenders:
+    rolling-file:
+      type: rolling-file
+      fileName: /var/logs/kibana.log
+      layout:
+        type: pattern
+      policy:
+        // ...
+      strategy:
+        // ...
+      retention:
+        maxAccumulatedFileSize: "5GB"
```

Comment:
Not needed, but do we have a reference for these "special" string format values?

For sure, mostly people know how they work (1MiB, 1MB, 1GB or 4h, 4d, 4w and so on), but could be nice to link to that reference from here (if it exists).

Reply:
Yeah, it would be very helpful. I will check other documented config settings that are using bytesize fields too, hopefully we have that documented somewhere.